e2e/runner/constants/exit-code.js

```diff
@@ -0,0 +1,2 @@
+export const SUCCESS_EXIT_CODE = 0;
+export const FAILURE_EXIT_CODE = 1;
```

Comment:
I'd really like to keep this change as the part of the PR

Reply:
doesn't offend me, could easily be a preliminary PR to make the diff nicer too